src/components/reportPage/BatchJob.jsx

```diff
@@ -1,70 +1,44 @@
-import React, { useState, useCallback, useEffect } from "react";
-import StepUpload from "./StepUpload";
-import StepEmail from "./StepEmail";
+import React, { useState } from "react";
+import UploadSection from "./UploadSection";
 import "./batch-job.css";
 
 const BatchJob = () => {
   const FILE_LIMIT = 100;
-  const [currentStep, setCurrentStep] = useState(1);
-  const [files, setFiles] = useState([]);
   const [email, setEmail] = useState();
-  const [emailVerified, setEmailVerified] = useState(false);
-  const [formData, setFormData] = useState({
-    email: "",
-    emailVerified: false,
-  });
-  const handleChange = (event) => {
-    // setFormData({
-    //   ...formData,
-    //   [event.target.name]: event.target.value,
-    // });
-  };
-
-  const handleEmail = async (e) => {
-    const { email } = e.target.elements;
+  const formId = "batchForm";
 
-    // Do some email verification
-
-    // Set email
-    setEmail(email.value);
-    setEmailVerified(true);
+  const handleChange = (event) => {
```

Comment:
Could you be more descriptive?

Reply:
Not sure what you mean. Do you mean the variable name "handleChange"?

Reply:
yup

Reply:
`const handleEmailChange = (event) =>`
Fixed it